### ISSUE TITLE
forwardport: rancher-post-delete workload missing / in image, causes fail to pull image

### ISSUE THREAD

**Issue by slickwarren** [rancher/rancher#32510]
forwardport of: rancher-post-delete workload missing / in image, causes fail to pull image rancher/rancher#32436

* deploy HA airgap setup

**Comment by deniseschannon** [comment]
Available to test in v2.6.0-rc1

**Comment by slickwarren** [comment]
tested on v2.6.0-rc1:
```
cattle-system         rancher-post-delete-x8kdh                Completed   0          6h50m
```
